app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -154,14 +154,20 @@ def mark_user_session_authenticated
   end
 
   def assign_variables_for_otp_verification_show_view
-    @phone_number = user_session[:unconfirmed_phone] ||
-                    decorated_user.masked_two_factor_phone_number
+    @phone_number = phone_to_deliver_to
     @code_value = current_user.direct_otp if FeatureManagement.prefill_otp_codes?
-    @context = context
     @delivery_method = params[:delivery_method]
     @reenter_phone_number_path = reenter_phone_number_path
   end
 
+  def phone_to_deliver_to
```

Comment:
In `devise/two_factor_authentication_controller.rb` this method always returns the actual phone number, never masked, what do you think about calling this something else, because it's for rendering? ex `display_phone_to_deliver_to` ?